src/lib/menu/menu-directive.ts

```diff
@@ -75,7 +75,10 @@ export class MdMenu implements AfterContentInit, MdMenuPanel, OnDestroy {
   @ContentChildren(MdMenuItem) items: QueryList<MdMenuItem>;
 
   /** Whether the menu should overlap its trigger. */
-  @Input() overlapTrigger = true;
+  @Input()
+  get overlapTrigger(): boolean { return this._overlapTrigger; }
+  set overlapTrigger(value: boolean) { this._overlapTrigger = coerceBooleanProperty(value); }
```

Comment:
`coerceBooleanProperty` should only be used for things that start as `false` and are optionally turned to `true`. 

@kara was `overlapTrigger` meant to be set to `true` by default?